Bugzilla/Bug.pm

```diff
@@ -302,6 +302,73 @@ with 'Bugzilla::Elastic::Role::Object';
 
 sub ES_TYPE {'bug'}
 
+sub ES_INDEX { Bugzilla->params->{elasticsearch_index} }
+
+sub ES_SETTINGS {
```

Comment:
Were these patterns copied as-is from their prior location, or were changes made in support of conveying them over?